Utility/roleManipulation.cc.lua

```diff
@@ -11,14 +11,12 @@ Usage:
 `moveroles`/`mroles` Take the roles from the first member and give them to the second member.
 */}}
 
-{{ if eq 2 (len .Message.Mentions) }}
-	{{ parseArgs 2 "You need to ping two people, do `-roles` for help." (carg "member" "1") (carg "member" "2") }}
-	{{ $user1 := getMember (index .CmdArgs 0) }}
-	{{ $user2 := getMember (index .CmdArgs 1) }}
-	{{ $list1 := $user1.Roles }} {{ $user1 := $user1.User }}
+{{ if eq (len .CmdArgs) 2 }}
```

Comment:
should be 
{{ if or (eq (len .Message.Mentions) 2) (eq (len (reFindAll `<@!?\d{17,19}>` .Message.Content)) 2) }}

this check for both mentions and IDs, not just any two thins seperated bya space
better yet, use parseArgs with member carg, member carg supports both id and mention already
 since we want two mentions, not just any two